### DISCUSSION TITLE
How to include hyperlinks in the TaskDialog?

### DISCUSSION THREAD

**Discussion by TuTAH1** [ookii-dialogs/ookii-dialogs-winforms#75] [Q&A] [answered]
The screenshot in the readme shows that I can use a hyperlinks. But how? Do I need to use some syntax?

readme.md seems to have not much info about this class library, just the description. Is there any documentation for this library anywhere? The documentation [here](http://www.ookii.org/docs/dialogs-1.0/) is not accessible

**Comment by augustoproiete** [comment, marked as answer]
Hi @TuTAH1, hyperlinks use the HTML syntax `<a href="..."></a>`

```csharp
taskDialog.EnableHyperlinks = true;
taskDialog.Footer = "Task Dialogs support <a href=\"https://augustoproiete.net\">hyperlinks</a>.";
```

You can see examples in the source-code:
https://github.com/ookii-dialogs/ookii-dialogs-winforms/blob/v4.0.0/sample/shared/MainForm.Designer.cs#L58-L61
